receiver/mysqlreceiver/metadata.yaml

```diff
@@ -552,6 +552,33 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: [cache_status]
+  mysql.qcache.hits:
+    enabled: false
+    description: The number of query cache hits.
+    unit: 1
+    sum:
+      value_type: int
+      input_type: string
+      monotonic: true
+      aggregation_temporality: cumulative
+  mysql.qcache.not.cached :
+    enabled: false
+    description: The number of noncached queries (not cacheable, or not cached due to the query_cache_type setting).
+    unit: 1
+    sum:
+      value_type: int
+      input_type: string
+      monotonic: true
+      aggregation_temporality: cumulative
+  mysql.qcache.queries:
+    enabled: false
+    description: The number of queries registered in the query cache.
+    unit: 1
+    sum:
+      value_type: int
+      input_type: string
+      monotonic: true
```

Comment:
This sounds like a "current" count, which would go up and down. If not, is it a running count of the number of queries which added to the cache at some point?

Reply:
I agree with you after giving it a second thought. I think you're correct, it is not monotonic.